javacutil/src/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -369,6 +369,11 @@ public static AnnotationMirror getSameIgnoringValues(
             new Comparator<AnnotationMirror>() {
                 @Override
                 public int compare(AnnotationMirror a1, AnnotationMirror a2) {
+                    // AnnotationMirror.toString() prints the elements of an annotation in the
```

Comment:
Do we have a way to also sort the elements for comparison? It could be confusing if the order of elements depends on the order of the annotation attributes.